### PR TITLE
[WIP] - Add API documentation and missing entries for MediaStream, MediaStreamTrack and MediaDevice

### PR DESCRIPTION
**Pull request by wer-mathurin**
Add missing property muted in MediaStreamTrack
Add missing callback handler for onUnMuted in MediaStreamTrack
Add missing API for MediaSteam(active and clone)

### CODE REVIEW

**Comment by cloudwebrtc** [comment]
Cool.

File: lib/src/native/media_stream_track_impl.dart [hunk 1]
```diff
@@ -15,11 +17,14 @@ class MediaStreamTrackNative extends MediaStreamTrack {
...
   bool _enabled;
 
+  bool _muted = false;
+
   @override
   set enabled(bool enabled) {
     _channel.invokeMethod('mediaStreamTrackSetEnable',
         <String, dynamic>{'trackId': _trackId, 'enabled': enabled});
     _enabled = enabled;
+    _muted = enabled;
```
Comment (cloudwebrtc) [inline lib/src/native/media_stream_track_impl.dart:27]: I think it should be ` _muted = !enabled`;

**Review by cloudwebrtc — CHANGES_REQUESTED** [review]

File: lib/src/native/media_stream_track_impl.dart (hunk 1 above)
Comment (wer-mathurin) [reply lib/src/native/media_stream_track_impl.dart:27]: You're right

**Review by cloudwebrtc — COMMENTED** [review]
I think we need to add `switchCamera` to `Helper`, because this method is useful in `example` and `flutter-webrtc-demo`.

**Review by cloudwebrtc — APPROVED** [review]

**Comment by wer-mathurin** [comment]
I'm almost done for the switch camera.
Just testing and implementing in the example.

Works on Web.

But I'm not able to run on Android.
As soon I open any menu I got this error
I/org.webrtc.Logging(13737): NativeLibrary: Loading native library:
jingle_peerconnection_so
I/org.webrtc.Logging(13737): NativeLibrary: Loading library:
jingle_peerconnection_so
E/org.webrtc.Logging(13737): NativeLibrary: Failed to load native library:
jingle_peerconnection_so
E/org.webrtc.Logging(13737): NativeLibrary: java.lang.UnsatisfiedLinkError:
dlopen failed: library "libjingle_peerconnection_so.so" not found
E/org.webrtc.Logging(13737): NativeLibrary: java.lang.UnsatisfiedLinkError:
dlopen failed: library "libjingle_peerconnection_so.so" not found

**Review by cloudwebrtc — APPROVED** [review]

**Comment by cloudwebrtc** [comment]
LGTM

**Pull request merged**